x-pack/plugins/security_solution/public/detections/components/rules/data_view_selector/translations.tsx

```diff
@@ -30,3 +30,18 @@ export const DATA_VIEW_NOT_FOUND_WARNING_DESCRIPTION = (dataView: string) =>
         'Your data view of "id": "{dataView}" was not found. It could be that it has since been deleted.',
     }
   );
+
+export const DDATA_VIEW_ALERTS_ON_ALERTS_WARNING_LABEL = i18n.translate(
+  'xpack.securitySolution.detectionEngine.stepDefineRule.dataViewIncludesAlertsIndexLabel',
+  {
+    defaultMessage: 'Default security data view',
```

Comment:
Consider capitalizing, since it's referring to the product name of the Security app/solution:
```suggestion
    defaultMessage: 'Default Security data view',
```